youtubesearchpython/core/channel.py

```diff
@@ -48,28 +79,58 @@ def parse_response(self):
             "title": getValue(response, ["metadata", "channelMetadataRenderer", "title"]),
             "banners": getValue(response, ["header", "c4TabbedHeaderRenderer", "banner", "thumbnails"]),
             "subscribers": {
-                "simpleText": getValue(response, ["header", "c4TabbedHeaderRenderer", "subscriberCountText", "simpleText"]),
-                "label": getValue(response, ["header", "c4TabbedHeaderRenderer", "subscriberCountText", "accessibility", "accessibilityData", "label"])
+                "simpleText": getValue(response,
+                                       ["header", "c4TabbedHeaderRenderer", "subscriberCountText", "simpleText"]),
+                "label": getValue(response, ["header", "c4TabbedHeaderRenderer", "subscriberCountText", "accessibility",
+                                             "accessibilityData", "label"])
             },
             "thumbnails": thumbnails,
-            "availableCountryCodes": getValue(response, ["metadata", "channelMetadataRenderer", "availableCountryCodes"]),
+            "availableCountryCodes": getValue(response,
+                                              ["metadata", "channelMetadataRenderer", "availableCountryCodes"]),
             "isFamilySafe": getValue(response, ["metadata", "channelMetadataRenderer", "isFamilySafe"]),
             "keywords": getValue(response, ["metadata", "channelMetadataRenderer", "keywords"]),
             "tags": getValue(response, ["microformat", "microformatDataRenderer", "tags"]),
-            "views": getValue(metadata, ["viewCountText", "simpleText"]),
-            "joinedDate": getValue(metadata, ["joinedDateText", "runs", -1, "text"]),
-            "country": getValue(metadata, ["country", "simpleText"])
+            "views": getValue(metadata, ["viewCountText", "simpleText"]) if metadata else None,
+            "joinedDate": getValue(metadata, ["joinedDateText", "runs", -1, "text"]) if metadata else None,
+            "country": getValue(metadata, ["country", "simpleText"]) if metadata else None,
+            "playlists": playlists,
         }
 
+    def parse_next_response(self):
+        response = self.data.json()
+
+        self.continuation = None
+
+        response = getValue(response, ["onResponseReceivedActions", 0, "appendContinuationItemsAction", "continuationItems"])
+        for i in response:
+            if getValue(i, ["continuationItemRenderer"]):
+                self.continuation = getValue(i, ["continuationItemRenderer", "continuationEndpoint", "continuationCommand", "token"])
+                break
+            self.result["playlists"].append(self.playlist_parse(i))
```

Comment:
This doesn't work. This way, all further playlists only contain None.
```suggestion
            elif getValue(i, ['gridPlaylistRenderer']):
                self.result["playlists"].append(self.playlist_parse(getValue(i, ['gridPlaylistRenderer'])))
            # TODO: Handle other types like gridShowRenderer
```

Reply:
Hmm, it seems like I haven't done proper testing & printed out final results. Interesting...